products/jbrowse-web/src/Loader.tsx

```diff
@@ -426,13 +426,15 @@ const Renderer = observer(
           // rootModel.setHistory(
           //   UndoManager.create({}, { targetStore: rootModel.session }),
           // )
-
           // make some things available globally for testing e.g.
           // window.MODEL.views[0] in devtools
-          // @ts-ignore
-          window.MODEL = rootModel.session
-          // @ts-ignore
-          window.ROOTMODEL = rootModel
+          if (inDevelopment) {
+            // @ts-ignore
+            window.MODEL = rootModel.session
```

Comment:
Could we also add `window.SESSION = rootModel.session`? I feel like it might be nice to have a bit more intuitive name for the variable.

Reply:
That's a good idea.